rails/app/controllers/api/v1/report_users_controller.rb

```diff
@@ -76,8 +76,7 @@ def query(options, user)
       cohorts: Pundit.policy_scope(user, Admin::Cohort),
       runnables: Pundit.policy_scope(user, Portal::Offering)
                   .joins("INNER JOIN external_activities ON external_activities.id = portal_offerings.runnable_id")
-                  .where("portal_offerings.runnable_type = 'ExternalActivity' " +
-                         "AND external_activities.tool_id IN (?)", lara_tool_ids)
+                  .where("portal_offerings.runnable_type = 'ExternalActivity'")
```

Comment:
It looks like this is the only place that uses lara_tool_ids, so it seems good to delete its definition on line 72.